work_queue/src/work_queue.c

```diff
@@ -1670,9 +1670,9 @@ static work_queue_result_code_t get_result(struct work_queue *q, struct work_que
 	timestamp_t effective_stoptime = 0;
 	time_t stoptime;
 
-	//Format: task completion status, exit status (exit code or signal), output length, execution time, taskid
-	char items[4][WORK_QUEUE_PROTOCOL_FIELD_MAX];
-	int n = sscanf(line, "result %s %s %s %s %" SCNd64, items[0], items[1], items[2], items[3], &taskid);
+	//Format: task completion status, exit status (exit code or signal), output length, execution time, taskid, loop device
+	char items[5][WORK_QUEUE_PROTOCOL_FIELD_MAX];
+	int n = sscanf(line, "result %s %s %s %s %" SCNd64" %s", items[0], items[1], items[2], items[3], &taskid, items[4]);
```

Comment:
I'm not sure about modifying the result message here. Perhaps we can do it through enum rmonitor_errors?